### ISSUE TITLE
FATAL: An error occurred: Missing libportal InputCapture portal support

### ISSUE THREAD

**Issue by bennypowers** [input-leap/input-leap#1728]
### What happened?

I would like to use input-leap as a server on a GNOME Wayland Fedora Workstation 39, and as a client on a GNOME-Wayland Fedora Silverblue 39 workstation, as well as as a client on a macos installation. 

Input-leap remains stuck on "Input leap is starting"

### Version

From Git HEAD or commit (specify below)

### Git commit hash (if applicable)

004a54fb

### If applicable, where did you install Input Leap from?

I Installed the ei-enabled package from this COPR https://discussion.fedoraproject.org/t/ofourdan-input-leap-ei-enabled/90196

If i uninstall and use the non-copr version 

```
❯ sudo dnf remove input-leap && sudo dnf install input-leap-2.4.0^20230918git004a54fb-1.fc39
```

in that case I get the same error as above

### What OSes are you seeing the problem on? (Check all that apply)

Linux, macOS

### If applicable, are you using Wayland or X11?

Fedora 39 workstation, GNOME 45 Wayland

### What OS versions are you using?

Server: Fedora 39 workstation
Client1: Fedora 39 silverblue
Client2: Mac OS Sonoma

### Relevant log output

```shell
I got this output:


❯ input-leaps --debug DEBUG2 --no-daemon --name i --use-ei --config ~/.config/input-leap/default.conf
[2023-11-14T08:06:33] DEBUG: opening configuration "/home/bennyp/.config/input-leap/default.conf"
[2023-11-14T08:06:33] DEBUG: configuration read successfully
[2023-11-14T08:06:33] DEBUG1: starting server
[2023-11-14T08:06:33] DEBUG1: thread 0x00000002 entry
[2023-11-14T08:06:33] FATAL: An error occurred: Missing libportal InputCapture portal support

[2023-11-14T08:06:33] DEBUG1: caught cancel on thread 0x00000002
```

Then, running the GUI:

```
❯ input-leap --debug DEBUG2
```

I see in the input-leap log window

```
[2023-11-14T08:10:06] INFO: starting server
[2023-11-14T08:10:06] INFO: config file: /tmp/InputLeap.bnGzFW
[2023-11-14T08:10:06] INFO: log level: INFO
[2023-11-14T08:10:06] FATAL: An error occurred: Missing libportal InputCapture portal support
[2023-11-14T08:10:07] ERROR: process exited with error code: 1
[2023-11-14T08:10:07] INFO: detected process not running, auto restarting
```

I get the same output when using the `--use-ei` flag
```


### Any other information

_No response_

**Comment by bennypowers** [comment]
This was mostly solved by the comments here: https://discussion.fedoraproject.org/t/input-leap-fedora-39-gnome-wayland/95856/2

tl;dr (fedora 39): 

```sh
sudo dnf remove input-leap
sudo dnf copr enable ofourdan/input-leap-ei-enabled
sudo dnf install input-leap --repo copr:copr.fedorainfracloud.org:ofourdan:input-leap-ei-enabled
sudo dnf reinstall libportal --repo copr:copr.fedorainfracloud.org:ofourdan:input-leap-ei-enabled
```
